### ISSUE TITLE
PS Terminal Crashes: An error occurred while starting PowerShell Editor Services

### ISSUE THREAD

**Issue by James-Livingston** [microsoft/vscode#72348]
Issue Type: <b>Bug</b>

Open any .ps1 file, the terminal window will display the following error:
"
An error occurred while starting PowerShell Editor Services:

Unable to load one or more of the requested types. Retrieve the LoaderExceptions property for more information.
   at System.Reflection.RuntimeModule.GetTypes(RuntimeModule module)
   at System.Reflection.Assembly.GetTypes()
   at Microsoft.PowerShell.Commands.AddTypeCommand.LoadAssemblyFromPathOrName(List`1 generatedTypes)
   at Microsoft.PowerShell.Commands.AddTypeCommand.EndProcessing()
   at System.Management.Automation.CommandProcessorBase.Complete()
"

VS Code version: Code 1.33.1 (51b0b28134d51361cf996d2f0a1c698247aeabd8, 2019-04-11T08:27:14.102Z)
OS version: Windows_NT x64 10.0.14393

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-7300U CPU @ 2.60GHz (4 x 2712)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|15.85GB (8.02GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (13)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-markdownlint|Dav|0.26.0
prettier-vscode|esb|1.8.1
gc-excelviewer|Gra|2.1.32
r|Iku|1.0.7
puppet-vscode|jpo|0.18.0
vscode-cosmosdb|ms-|0.10.1
mssql|ms-|1.5.0
python|ms-|2019.3.6558
azure-account|ms-|0.8.2
powershell|ms-|2.0.2
r-lsp|REd|0.1.0
code-spell-checker|str|1.6.10
vscode-ansible|vsc|0.5.2

(2 theme extensions excluded)

</details>

**Comment by davesbrown** [comment]
Getting same error windows server 2016

**Comment by TimK4** [comment]
I had the same issue, went through a couple of different things and uninstalled/reinstalled, nothing worked until I uninstalled and reinstalled the PowerShell extension.  I believe it installed at a newer version - 1.12.1.

**Comment by brycesteel** [comment]
I had the same issue with the PowerShell Preview extension (ms-vscode.powershell-preview@2019.5.0, VS Code x64 1.34.0) on multiple Server 2016 systems with the out-of-the-box .NET 4.7.0.  The error went away and things started working as soon as I installed .NET 4.7.2.  I am _assuming_ there is a dependency / prerequisite of .NET 4.7.1 (given I had no such problems on Win10 1709 with .NET 4.7.1), but I did not separately test that.  It would be nice if the extension requirements / "Platform support" mentioned this apparent requirement.  BTW - loving the PSReadLine integration!
